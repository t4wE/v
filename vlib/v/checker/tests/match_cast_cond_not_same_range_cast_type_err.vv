const (
	a = u16(1)
	b = u16(5)
)
c := u8(3)
match c {
	a...b { println('1...5') }
	else { println('not 1...5') }
}
