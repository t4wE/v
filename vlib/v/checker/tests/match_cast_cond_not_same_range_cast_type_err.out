vlib/v/checker/tests/match_cast_cond_not_same_range_cast_type_err.vv:7:2: error: the range type and the match condition type should match
    5 | c := u8(3)
    6 | match c {
    7 |     a...b { println('1...5') }
      |     ~~~~~
    8 |     else { println('not 1...5') }
    9 | }
Details: 
match condition type: u8
          range type: u16
